Bugfix/edits and restructuring (#62)
chec/commerce.js: docs/00_Getting_started.md

import Commerce from '@chec/commerce.js';

// Create a Commerce instance
const commerce = new Commerce('{public_api_key}');
```

<div class="highlight highlight--note">
    <span>Note</span>

>     <p>We've built in a console debugger into the Commerce.js SDK to help with debugging during development . To enable the debugger you can include the second argument <code>true</code> when you create your Commerce instance like so: <code>const commerce = new Commerce('{public_api_key}', true);</code>. Note that only a test API key has to be provided as the first argument and not the live API key for the console to show messages.</p>

Reviewer: ```suggestion
    <p>We've built in a console debugger into the Commerce.js SDK to help with debugging during development. To enable the debugger you can include the second argument <code>true</code> when you create your Commerce instance like so: <code>const commerce = new Commerce('{public_api_key}', true);</code>. Note that a test API key has to be provided as the first argument for the console to show messages - it won't work with a live key.</p>
```
